tox.ini

```diff
@@ -88,7 +88,7 @@ commands =
     sh -c 'stat promql-transform > /dev/null 2>&1 || curl "https://github.com/canonical/promql-transform/releases/download/2.25.2-1rc1/promql-transform_2.25.2-1rc1_linux_amd64.tar.gz" -L -s | tar zxv promql-transform'
     coverage run \
       --source={[vars]src_path},{[vars]lib_path} \
-      -m pytest -v --tb native -s {posargs} {[vars]tst_path}/unit
+      -m pytest -v --tb native --log-cli-level=INFO -s {posargs} {[vars]tst_path}/unit
```

Comment:
Intentional or debugging?

Reply:
I believe that is intentional @sed-i ?

Reply:
Yes that is intentional and in line with other charms, though obviously not necessary.